cmake/Modules/LocalAom.cmake

```diff
@@ -1,5 +1,5 @@
 set(AVIF_AOM_GIT_TAG v3.12.1)
-set(AVIF_AVM_GIT_TAG research-v9.0.0)
+set(AVIF_AVM_GIT_TAG 95163cd4a3c3a4714985e7f71ff021ffe096afd9)
```

Comment:
There are test failures that prevent us from going further than this commit: https://gitlab.com/AOMediaCodec/avm/-/issues/717